etc/igraph.sh

```diff
@@ -0,0 +1,12 @@
+wget http://switch.dl.sourceforge.net/sourceforge/igraph/igraph-0.5.4.tar.gz
```

Comment:
We can probably invoke this script inside `travis_installs.sh` and `install_deps.sh`